buildSrc/src/main/java/org/springframework/gradle/github/release/RepositoryRef.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2002-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.gradle.github.release;
+
+/**
+ * Steve Riesenberg
+ */
+public class RepositoryRef {
```

Comment:
This looks like it's the same class as `org.springframework.gradle.github.milestones.RepositoryRef`. Could we extract it into `org.springframework.gradle.github` and reuse it in both `milestones` and `release` packages?